libs/image-utils/src/image_data.ts

```diff
@@ -96,38 +101,103 @@ export function fromGrayScale(
   return createImageData(imageDataBuffer, width, height);
 }
 
+function createCanvasWithImageData(imageData: ImageData) {
+  const canvas = createCanvas(imageData.width, imageData.height);
+  const context = canvas.getContext('2d');
+  context.putImageData(ensureImageData(imageData), 0, 0);
+  return canvas;
+}
+
 /**
  * Creates a data URL from image data.
  */
 export function toDataUrl(
   image: ImageData,
   mimeType: 'image/png' | 'image/jpeg'
 ): string {
-  const { width, height } = image;
-  const canvas = createCanvas(width, height);
-  const context = canvas.getContext('2d');
-  context.putImageData(image, 0, 0);
+  const canvas = createCanvasWithImageData(image);
   return mimeType === 'image/jpeg'
     ? canvas.toDataURL(mimeType)
     : canvas.toDataURL(mimeType);
 }
 
 /**
  * Writes an image to a file.
+ *
+ * @param buffer - Whether to buffer the image before writing it to disk.
```

Comment:
I don't see a `buffer` param - is this outdated?

Reply:
Ah yep. Will remove 👍